### ISSUE TITLE
3.3 - (AI) Civ uses a GD on a CS in the very turn another civ acquires a SOI

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#9752]
_1. Mod Version (X.Y.Z). Current Version: 3.3.1_

3.3

_2. DirectX Version_

_3. List of Other Mods_

[ModList2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/11132883/ModList2.zip)

_4. Describe the Issue_

Ethiopia used a great diplomat on Mogadishu, but on the very same turn I just secured my sphere on it. It should have waited 1 turn to see if the vote passed my way, then either use the GD if I didn't get the vote or move the GD somewhere else.
______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

[Sid Meier's Civilization 5.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/11132882/Sid.Meier.s.Civilization.5.zip)

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

[Uploading Logs - Copy.zip…]()

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_
![image](https://user-images.githubusercontent.com/20593976/229368878-a4172e70-b0d3-4ed1-886d-a2723b2412cf.png)

**Comment by azum4roll** [comment]
Well this is an edge case. The proposal hasn't passed when he used the Great Diplomat.

**Comment by Stalker012345** [comment]
> Well this is an edge case. The proposal hasn't passed when he used the Great Diplomat.

exactly, but a smart human wouldn't burn a GD on a CS that will literally flip to another civ right after the GD expenditure, so the right play is to wait 1 turn and see. I don't consider it a bug, but it is an AI improvement that could be considered.
